pkgs/servers/sql/proxysql/default.nix

```diff
@@ -67,9 +67,7 @@ stdenv.mkDerivation rec {
     zlib
   ];
 
-  enableParallelBuilding = true;
```

Comment:
why?

Reply:
This: https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/unnecessary-parallel-building.md

Reply:
but this isn't built with any of those?

Reply:
Yes, but this project isn't built with it :D

It's built with a regular `Makefile`. 

And the setup hook linked in your explanation doesn't (or shouldn't) get run, because of https://github.com/NixOS/nixpkgs/blob/ba8786625857cee5d4d1b4b7baed5cf42708b789/pkgs/servers/sql/proxysql/default.nix#L72

This package is… bad, but I blame upstream.

Reply:
allrighty, let me revert that change.

Reply:
Job done.